secret_hitler.py

```diff
@@ -469,6 +469,22 @@ def check_name(self, name, current_player=None):
             return "Stop being a 👹 Pick a shorter name!"
         return None
 
+    def join_game_timeout(self, args, from_player):
+        """
+        You can join a game but specify a window of availability... if the game doesn't start after N minutes your willingness to join will be forcibly revoked
+        Also resets expiry if they set a new time
+        """
+        if args:
+            try:
+                joinExpiry = int(args)
+                expiryTime = time.time() + joinExpiry * 60
+                from_player.joinTimeout = expiryTime
+                return "\nYou have set a join timeout for {} minute{} ⌚️👀".format(joinExpiry,"s" if joinExpiry > 1 else "")
+            except ValueError as e:
+                #Ignore invalid time
+                return "\n/leavegame then /joingame **NUMBER** if you want your join to expire after X Minutes *Can't wait around all day for a game right?!*"
+
```

Comment:
```suggestion
        return ""
```

Reply:
Do we not need this?

Reply:
This change should just be to add a **return ""** in the case that no args are supplied to the function.
Otherwise on line 1140 it's += the result of this function which wouldn't be returning anything in that case.